src/coreclr/src/jit/emit.cpp

```diff
@@ -6422,7 +6422,11 @@ unsigned char emitter::emitOutputLong(BYTE* dst, ssize_t val)
 
 unsigned char emitter::emitOutputSizeT(BYTE* dst, ssize_t val)
 {
+#if !TARGET_64BIT
+    MISALIGNED_WR_I4(dst, (int)val);
```

Comment:
This seems wrong. It seems like we should rename `emitOutputSizeT` to `emitOutputTargetSizeT` and have it take a `target_ssize_t`.

Reply:
Eh, I'd prefer naming changes be done by the jit team for the jit team in a separate pr. All of the emit functions take ssize_t, regardless of the size of the generated bits.